build.gradle

```diff
@@ -22,7 +22,19 @@ dependencies {
 
 mainClassName = 'com.gradle.exportapi.Application'
 
+task validateDbConfig {
+    description 'Ensure that the target database properties file exists'
+    group 'Verification'
+
+    ext.configuration = file('src/main/resources/POSTGRES.properties')
+    doLast {
+        assert configuration.exists() : "Please define target database information in $configuration"
```

Comment:
I wasn't aware of this syntax for doing assertions in Groovy/Gradle. I like it. Thanks for teaching me a thing!